core/interfaces/i_toolbox_item.js

```diff
@@ -89,4 +89,12 @@ IToolboxItem.prototype.dispose;
  */
 IToolboxItem.prototype.getClickTarget;
 
+/**
+ * Sets whether the category is visible or not.
+ * For a category to be visible its parent category must also be expanded.
+ * @param {boolean} isVisible True if category should be visible.
+ * @protected
+ */
+IToolboxItem.prototype.setVisible_;
```

Comment:
Is this supposed to exist here? Or on the collapsible toolbox item interface?

Reply:
It has to move all the way up to IToolboxItem because of how it's being used. @alschmiedt said that "we allow any toolbox item to be in a collapsible category" and that it should be left as is for compatibility reasons.